nextjs-13/src/app/page.tsx

```diff
@@ -15,12 +20,29 @@ const Home = () => {
   }
 
   return (
-    <>
+    <div className="home">
       <Head>
         <title>Welcome to NextNewsletter 🚀</title>
       </Head>
-      <h4>Welcome to NextNewsletter 🚀</h4>
-    </>
+      <div className="home__nav">
+        <div className="home__tab">Newsletter</div>
+      </div>
+      <div className="home__dashboard">
+        <div></div>
```

Comment:
I think you can remove this empty `div`

Reply:
Actually I need that `div` there.

It's because I want the [`Create Newsletter`](https://github.com/nimblehq/js-framework-benchmark/pull/106/files#diff-223e9b714cf2c9b5f163079f0bd63d3a6eac9c46915fc3afde20a2742897d706R32-R38) button to be at the utmost right, like this:

![CleanShot 2023-06-06 at 13 32 18@2x](https://github.com/nimblehq/js-framework-benchmark/assets/76901736/7668c9c5-06e0-47b0-8dee-fafb98fd59d9)

And I think that I should use `display: flex` to do this, so I'm using the parent(the one with className `home__dashboard`) [with `display: flex` and `justify-content: space-between`](https://github.com/nimblehq/js-framework-benchmark/pull/106/files#diff-e8ab88801c53a9530e8eca7be803e279c44fa9d639d72b594a0b809a24bcfad4R30), and putting the empty div there so that it be at the utmost left, and the [`Create Newsletter`](https://github.com/nimblehq/js-framework-benchmark/pull/106/files#diff-223e9b714cf2c9b5f163079f0bd63d3a6eac9c46915fc3afde20a2742897d706R32-R38) button at the utmost right.

![CleanShot 2023-06-06 at 13 32 18@2x](https://github.com/nimblehq/js-framework-benchmark/assets/76901736/23be9d51-d785-4321-ae7c-5107009e123a)

Anyway, I [changed it from `<div></div>` to `<div />`](https://github.com/nimblehq/js-framework-benchmark/pull/106/commits/b5c4314bf5475947150344015ba6bcdc399813cc#diff-223e9b714cf2c9b5f163079f0bd63d3a6eac9c46915fc3afde20a2742897d706R31) to looks a bit more concise 👀

Reply:
Okie understood, that's better

Reply:
Wait, TIL that `flex-end` exists 🤣

Improved [here](https://github.com/nimblehq/js-framework-benchmark/pull/106/commits/a41562135f8d8a1b30e2e0e1025831022f64e27a).